Makefile

```diff
@@ -1,14 +1,14 @@
-ifndef $(GOPATH)
+ifndef GOPATH
     GOPATH=$(shell go env GOPATH)
     export GOPATH
 endif
 
-ifndef $(HOST_ADDRESS)
+ifndef HOST_ADDRESS
     HOST_ADDRESS=$(shell dig @resolver4.opendns.com myip.opendns.com +short)
     export HOST_ADDRESS
 endif
 
-ifndef $(BUILD_PATH)
+ifndef BUILD_PATH
     BUILD_PATH="/go/bin/ssvnode"
```

Comment:
```suggestion
    BUILD_PATH="./bin/ssvnode"
```

Reply:
@pavelkrolevets I think this might break it for people who use `docker run <...> make start-node`